content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+    You must be prepared to deal with bugs and issues and a high hardware requirement of a (recommended) minimum of 16 CPU cores.
+    Consider using other (more stable) softwares such as [Pufferfish](#pufferfish-recommended) or [Purpur](#purpur) beforehand.
+
+    **Ensure you thoroughly test every feature, command and mechanic in your server while using this software.**
+
+    Additionally, **Folia does not fix lag**, and requires a very specific playerbase where everyone is spread out relatively far.
+    If your players are not spread out then there will be more entities in each region, which will cause more lag, making the experience identical if not worse than using a software such as Paper.
+
+</Callout>
+
+<Card href="https://jar.setup.md/download/folia/latest/latest" title="Download">
+  Download a copy of the latest Folia server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## MultiPaper
+
+[MultiPaper](https://github.com/MultiPaper/MultiPaper) is a 1.20.1 fork of [Purpur](#purpur) that splits your world(s) into multiple servers, running independently.\
+Each world is split into regions (similar to Folia), and each region runs on its own server.\
+This server software is often known as "sharding", where each server is a "shard".\
+Each server communicates with each other and is able to transfer players between servers, allowing for a seamless experience.
+
+<Callout type="warn" title="Warning">
+    **MultiPaper is highly unstable!**\
+    You must consider whether you need to use it. MultiPaper is notorious for being unstable and is not recommended for production use.\
+    Although many public servers such as DonutSMP use a sharding solution, they are often custom-made and are not available to the public.
+
+    Consider using / testing other (more stable) softwares such as [Pufferfish](#pufferfish-recommended) or [Purpur](#purpur) beforehand.
+    **Ensure you thoroughly test every feature, command and mechanic in your server if you decide to use this software.**
+</Callout>
+
+<Card href="https://multipaper.io/download.html" title="Download">
+  Download a jarfile for MultiPaper here.
+</Card>
+
+## 1.8 Softwares
+There are some spigot or paper forks that are actively maintained for 1.8.x.\
+We would recommend against using 1.8 or arguably any version below latest, due to the presence of security vulnerabilities or other exploits.\
+One of the main reasons for choosing to run a 1.8.x fork is for good performance on older hardware, or for pre-1.9 PvP.\
+See below for some 1.8.x forks that are actively maintained:
+
+<Cards>
+    <Card href="https://github.com/hpfxd/PandaSpigot" title = "PandaSpigot"></Card>
+    <Card href="https://github.com/Wind-Development/WindSpigot" title = "WindSpigot"></Card>
+    <Card href="https://github.com/CobbleSword/NachoSpigot" title = "NachoSpigot"></Card>
+</Cards>
+
```

Comment:
I think it's valuable to have the modded server software more clearly divided from the Bukkit derivative sections. Might need some minor layout tweaking elsewhere to make it consistent throughout, but I'm sure you get the idea. Modded versus plugin is a pretty major category difference.

Reply:
This was more or less done in [bf3319b](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/bf3319b76bf41d657ef572b076edcff7516ae2b6)